fixed skip empty file logic

ryancrawcour opened the pull request:
changed content_length != 0 to content_length > 508
content_length == 508 is an empty file, so only process content_length > 508 i.e. skip  empty files

azurecla commented:
Hi __@flanakin__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (micflan). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

deneha commented:
@flanakin Thanks for this contribution.  You'll want to put your GitHub username "flanakin" in the "authors" field instead of your name.

flanakin commented:
Ah, thank you! That explains why my alias didn't work :-P

ryancrawcour committed:
fixed skip empty file logic

changed content_length != 0 to content_length > 508
content_length == 508 is an empty file, so only process content_length > 508 i.e. skip  empty files
